### PR TITLE
Signup dev complete

### PR DESCRIPTION
**Pull request by simrin051**
Files To Check 

1. signup.html
2. signup.css

Mobile view

image

Desktop view

image

### CODE REVIEW

File: signup.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+.Signup:hover {
+    background-color: var(--primary-color);
+    color: white;
+}
+
+.Signup {
+    display: block;
+    background-color: black;
```
Comment (sri-rishi) [inline signup.css]: Suggestion: use variable color naming for colors

File: signup.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    box-shadow: 13px 13px 0px 0px #ffffff;
+}
+
+.options {
+    font-size: small;
+}
+
+.authenticate {
+    display: flex;
+
```
Comment (sri-rishi) [inline signup.css:48]: Suggestion: remove extra spacing in file

File: signup.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title"><i class="fa fa-wpforms fa-2x" aria-hidden="true"></i>CREATE NEW ACCOUNT</h5>
+                </div>
+                <div class="modal-body">
+                    <form>
+                        <div class="form-group">
+                            Name <span class="required-asterisk">*</span>
+                            <input type="name" placeholder=" " class="form-control rounded-0" id="userName" required>
```
Comment (sri-rishi) [inline signup.html:21]: Suggestion: use `label` tag with input or provide `aria-label` attribute in `input` tag for accessibility

**Review by sri-rishi — COMMENTED** [review]
Code look good to merge

**Comment by simrin051** [comment]
@sri-rishi 

Thanks for reviewing my PR. I have resolved the comments. Kindly seek your approval.

**Pull request merged**